woocommerce/woocommerce-android
# Create Order detail address editing actions

Summary
==========
Fixes issue #4886 by creating two ImageButtons as edit actions for both shipping and billing addresses and set the visibility to be controlled by the Order Editing feature flag.

Screenshots
==========
| Before  | After |
| ------------- | ------------- |
| ![Screenshot_20211006_070216](https://user-images.githubusercontent.com/5920403/136183158-ae445e65-7774-478c-affa-2c05d3235940.png) | ![Screenshot_20211006_065534](https://user-images.githubusercontent.com/5920403/136183139-43def0c5-b609-4fa4-9495-da23c282494a.png) |

How to Test
==========
1. Create an Order containing at least the shipping AND billing addresses descriptions
2. Go to the Order Details of the selected Order
3. Verify if both pencil icons appear to Shipping and Billing

Update release notes:
